test/server/error-formatter-test.js

```diff
@@ -99,18 +99,19 @@ describe('Error formatter', function () {
     describe('Assertions', function () {
         it('Should format "eq" assertion message', function () {
             var err = {
-                stepName:          'Step',
-                expected:          '"12345678901"',
-                actual:            '"00000000000"',
-                relatedSourceCode: 'eq(["12345678901"], ["00000000000"])',
-                key:               0,
-                isArrays:          true,
+                stepName:          'Step with <tag>',
+                expected:          '"<another-tag>"',
+                actual:            '"<some-tag>"',
+                relatedSourceCode: 'eq({"<tag>": "<some-tag>"}, {"<tag>": "<another-tag>"})',
+                key:               '<tag>',
+                isObjects:         true,
                 type:              TYPE.eqAssertion,
                 userAgent:         userAgentMock,
+                screenshotPath:    '/unix/path/with/<tag>',
```

Comment:
I've missed that assertions can have a message, we need to test it too. So I'll add a `message` field with some text (and `<tags>`, of course :P).